File: eco-core/core-backend/src/main/java/com/willfp/eco/internal/config/json/EcoJSONConfigSection.java
package com.willfp.eco.internal.config.json;

import org.jetbrains.annotations.NotNull;

import java.util.Map;

public class EcoJSONConfigSection extends EcoJSONConfigWrapper {
    public EcoJSONConfigSection(@NotNull final Map<String, Object> values) {
        this.init(values);
    }
}

File: eco-core/core-backend/src/main/kotlin/com/willfp/eco/internal/config/json/EcoJSONConfigSection.kt
package com.willfp.eco.internal.config.json

import com.willfp.eco.core.placeholder.StaticPlaceholder

@Suppress("UNCHECKED_CAST")
class EcoJSONConfigSection(values: Map<String, Any?>, injections: Collection<StaticPlaceholder> = emptyList()) : EcoJSONConfigWrapper() {
    init {
        init(values)
        this.injections = injections.toMutableList()
    }
}